resources/views/components/menu/account.blade.php

```diff
@@ -1,49 +1,76 @@
-{{--@guest
+<?php
```

Comment:
I agree and there certainly is! The prototype went quite a few steps farther than this and it was planned to remove the card eventually. 
We can have this right away, yet, while working on this I realized that it would blow up this PR quite a bit though because it requires some additional changes.
I think it would be best for me to follow up with another PR to address/discuss those changes separately?

Reply:
What about just removing the icon and points from the top widget so it's just username, mail, and tickets?

Reply:
That works for now, too - done in https://github.com/RetroAchievements/RAWeb/pull/1776/commits/1e2bde229b1f399ed5f7186e353ad77aef9fac4d

![Screenshot 2023-08-27 at 13 04 26](https://github.com/RetroAchievements/RAWeb/assets/1280590/361332d6-79a4-4108-bb89-5d9ef20cf3d2)

Reply:
Heads up: this is what it looks like with the remaining elements moved to the navbar:
![Screenshot 2023-08-27 at 13 08 55](https://github.com/RetroAchievements/RAWeb/assets/1280590/3cab52ec-a99f-4423-b8fe-5acc2f4aa127)
![Screenshot 2023-08-27 at 13 09 01](https://github.com/RetroAchievements/RAWeb/assets/1280590/dfae585f-3f1f-43f7-9bab-978286820397)

Allows us to drop the top card, make the brand logo more prominent, or get rid of the whole top brand area altogether (be it just on pages except on home, or on home, too)